### PR TITLE
Sidebar consistency

### PR DESCRIPTION
**Pull request by scharrier**
This PR moves "Markdown support" and "References" under the "Specification support" folder to be consistent with the existing "Polymorphism" page added there, and the existing "OpenAPI support" and "AsyncAPI support".

Capture d’écran 2023-02-06 à 17 24 54

I also added the URL redirects, so old links still work.

@jayk-u, does that make sense to you?

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -1,2 +1,5 @@
 https://help.bump.sh/* https://docs.bump.sh/help/:splat/ 301!
 / /help/ 302
+
+https://help.bump.sh/markdown-support https://docs.bump.sh/help/specifications-support/markdown-support/
+https://help.bump.sh/references https://docs.bump.sh/help/specifications-support/references/
```
[the remark this answers is not in the document]
Comment (scharrier) [reply static/_redirects:5]: 🤦 Thanks

[the remark this answers is not in the document]
Comment (paulRbr) [reply static/_redirects:2]: Please note: I have a doubt whether this redirects file works (I'm debugging right now)

**Review by paulRbr — APPROVED** [review]
LGTM

**Review by jayk-u — APPROVED** [review]
Totally okay to me

**Pull request merged**